CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+project(CMaizeSuperbuild LANGUAGES CXX)
+
+list(APPEND BOOST_COMPONENTS_REQUIRED filesystem system)
```

Comment:
For now I recommend having no dependencies in this project. Once your superbuild works for your hello world library then you can make the library depend on CMaizeExamples/CMakePublic. I wouldn't add any other dependencies to the project beyond that.